addons/fix-uploaded-svgs/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "SVG uploads fix",
-  "description": "Fixes a bug in the Scratch editor that makes uploaded SVG files created with some image editors, such as Affinity Designer, broken.",
+  "description": "Fixes a bug in the Scratch editor that breaks some uploaded SVG files, such as those created with Affinity Designer or generated by scratchblocks.",
```

Comment:
Do we want to be specifically mentioning third-party software? Do we know what platforms' SVGs experience this bug? I know this is somewhat unrelated, but now is a good time to maybe revise this.

Reply:
I think it's fine when put certain ways like "such as Affinity Designer", but "such as those created with Affinity Designer" way of wording it makes it feel a little more like free advertising. It's probably a matter of how specifically the product is being mentioned - "such as" is further away from the product names in the proposed description.

Reply:
Maybe this should be changed to "Fixes some bugs in the Scratch editor that break uploaded SVG files from some image editors, such as Affinity Designer or scratchblocks"?